src/fluree/db/json_ld/transact.cljc

```diff
@@ -18,6 +18,7 @@
             [clojure.core.async :as async]
             [fluree.db.json-ld.credential :as cred]
             [fluree.db.policy.enforce-tx :as policy]
+            [fluree.db.json-ld.policy :as perm] ;;TODO
```

Comment:
this is pretty minor, but could you elaborate on what is left to be done here?

Reply:
Whoops, this was probably some weird copypasta shenanigans, I'll remove it